twist_exome_38_template.sbatch

```diff
@@ -5,16 +5,16 @@
 #SBATCH -N 1
 #SBATCH -n 1
 #SBATCH -t 10-00:00:00
-#SBATCH -o "/lupus/proj_nobackup/ngi2016001/NGI/ANALYSIS/=PROJECT=/TwistExome.%j.out"
-#SBATCH -e "/lupus/proj_nobackup/ngi2016001/NGI/ANALYSIS/=PROJECT=/TwistExome.%j.err"
-#SBATCH -D "/lupus/proj_nobackup/ngi2016001/NGI/ANALYSIS/=PROJECT=/"
+#SBATCH -o "/proj/ngi2016001/nobackup/NGI/ANALYSIS/=PROJECT=/TwistExome.%j.out"
```

Comment:
There are a whole lot of paths in this file that do not exist. My opinion is that we should just remove these scripts; they still exist in the history if someone should need them. If you don't want to remove them, I suggest we at least leave them as is to make it clear that they have not been updated/tested to work on miarka.

Reply:
I think this script is needed, I'll review the paths again and see if I can fix

Reply:
I only found one path that I changed, the other are paths that are created in the process when the script is used or reference data that should be moved to the place pointed to in the script.